images/hyperkube/Dockerfile.rhel

```diff
@@ -2,10 +2,10 @@ FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
 WORKDIR /go/src/github.com/openshift/origin
 COPY . .
 RUN for p in vendor/k8s.io/kubernetes/cmd/kube-apiserver vendor/k8s.io/kubernetes/cmd/kube-controller-manager \
-    vendor/k8s.io/kubernetes/cmd/kube-scheduler vendor/k8s.io/kubernetes/cmd/kubelet; do make build WHAT=$p; done && \
+    vendor/k8s.io/kubernetes/cmd/kube-scheduler vendor/k8s.io/kubernetes/cmd/kubelet cmd/watch-termination; do make build WHAT=$p; done && \
```

Comment:
make sure you don't have to update `hyperkube` and `rpm`, see https://github.com/openshift/origin/pull/24275

Reply:
We don't need this in the RPM. Do you think I have to do anything beyond this here?

Reply:
From #24275 I think we are fine.